Makefile

```diff
@@ -287,6 +299,8 @@ docker run -dt --privileged \
 	--add-host "$(REGISTRY_HOST):$(shell $(IP_CMD) $(MASTER_CTR)1 2>/dev/null || echo 127.0.0.1)" \
 	$(DOCKER_IMAGE);
 sleep 2;
+docker exec $(1)$(2) mkdir -p /var/lib/dcos
```

Comment:
I see you were just following the existing pattern. Tho when you added a `/bin/bash -c` line it made the pattern look kinda silly. Plus, when you use `/bin/bash -c` you lose the error checking. You're probably want to do `/bin/bash -ce` or go multi-line and use `set -o errexit -o nounset -o pipefail`.